package-lock.json

```diff
@@ -1582,7 +1582,7 @@
     "browserslist": {
       "version": "2.11.0",
       "resolved": "https://registry.npmjs.org/browserslist/-/browserslist-2.11.0.tgz",
-      "integrity": "sha512-mNYp0RNeu1xueGuJFSXkU+K0nH+dBE/gcjtyhtNKfU8hwdrVIfoA7i5iFSjOmzkGdL2QaO7YX9ExiVPE7AY9JA==",
+      "integrity": "sha1-UDUNaHOoLr4POuVINljFca5fnX0=",
```

Comment:
I'm a little concerned about the size of the package-lock diff here. Why did all these `integrity` hashes update from sha512 to sha1? Are we going to have this amount of thrash with every PR?